controller/token.go

```diff
@@ -346,36 +346,37 @@ func (c *TokenController) Delete(ctx *app.DeleteTokenContext) error {
 	return ctx.OK([]byte{})
 }
 
-// Exchange provides OAuth2 token exchange. Currently only grant_type="client_credentials" is supported
-// allowing clients to authenticate using a service account ID and secret value.
+// Exchange provides OAuth2 and OpenID Connect token exchange.
+// Currently only grant_type="client_credentials", "authorization_code", and "refresh_token" are supported.
+//
+// grant_type="client_credentials" allows clients to authenticate using a service account ID and secret value.
 // A service account token is returned as the result of successful exchange.
```

Comment:
We should also remove this line, since it doesn't return just service account token any more

Reply:
This part of the comment is for client_credentials only.

Reply:
Oh yes, my bad. No problem in keeping it then